include/mbedtls/md.h

```diff
@@ -112,6 +112,9 @@
 #define MBEDTLS_MD_CAN_SHA512
 #define MBEDTLS_MD_SOME_LEGACY
 #endif
+#if defined(MBEDTLS_SHA3_C)
+#define MBEDTLS_MD_CAN_SHA3
```

Comment:
I'm not sure that's quite right. PSA has distinct algorithm identifiers for the various SHA-3 variants, hence distinct `PSA_WANT` macros. Since the `MD_CAN` macros are here to provide an abstraction above `MBEDTLS_xxx_C` and `PSA_WANT` macros, they need to be as fine-grained as the most fine-grained of the two, so PSA here.

Note: since this is a public macro, this needs fixing before the next release.

Reply:
Created https://github.com/Mbed-TLS/mbedtls/issues/7713 and added it to the next release EPIC.